app/ide-desktop/lib/dashboard/src/modals/ProjectLogsModal.tsx

```diff
@@ -53,8 +55,20 @@ function ProjectLogsModalInternal(props: ProjectLogsModalInternalProps) {
   })
 
   return (
-    <pre className="relative overflow-auto whitespace-pre-wrap">
-      <code>{logsQuery.data}</code>
-    </pre>
+    <div className="flex flex-col">
+      <div className="flex h-row items-center gap-4 self-start rounded-full border-0.5 border-primary/20 px-[11px] text-primary/50">
```

Comment:
let's remove the hardcoded height here. `text-primary/50` also looks unnecessary.